DuckDuckGo/SpecialErrorPage/SpecialErrorPageNavigationHandler+MaliciousSite.swift

```diff
@@ -64,6 +64,7 @@ final class MaliciousSiteProtectionNavigationHandler {
     @MainActor private(set) var maliciousURLExemptions: [URL: ThreatKind] = [:]
     @MainActor private(set) var bypassedMaliciousSiteThreatKind: ThreatKind?
     @MainActor private(set) var maliciousSiteDetectionTasks: [URL: Task<MaliciousSiteProtectionNavigationResult, Never>] = [:]
+    private var lastMainFrameNavigationURL: URL?
```

Comment:
actually we could directly check `maliciousSiteDetectionTasks[url]` and drop this extra variable